static/src/javascripts/projects/common/modules/crosswords/clues.js

```diff
@@ -90,13 +90,15 @@ const Clues = createClass({
 
     scrollIntoView(clue: Object) {
```

Comment:
Sad that this module isn't using the [Clue type](https://github.com/guardian/frontend/blob/master/static/src/javascripts/__flow__/types/crosswords.js#L35-L46). `id` should not be nullable, so I'm not sure what's going wrong here